src/coreclr/clr.featuredefines.props

```diff
@@ -38,7 +38,6 @@
 
     <!-- Features we're currently flighting, but don't intend to ship in officially supported releases -->
```

Comment:
You can delete this. We have runtimelab for this now.

Reply:
Good point, removed in 3rd commit.